### ISSUE TITLE
information profile

### ISSUE THREAD

**Issue by missfx** [daggerhart/openid-connect-generic#4]
It's possible to have te access token request in the headers rather than the url ? 
because I can not retrieve profile information, any ideas ? 
thanks a lot

**Comment by daggerhart** [comment]
I think we can do that.  I'll look at the specs to see if there is a preferred header for that type of thing.  If not, I'll make something up

**Comment by wgengarelly** [comment]
Thanks for improving the original plugin!

I also ran into an issue when the access token was included in the url. If I'm reading the spec correctly,  [section 5.3.1](http://openid.net/specs/openid-connect-core-1_0.html#UserInfoRequest) recommends sending the access token in the Authorization header field. My identity provider (IdentityServer3) seems to require the access token in the header, and doesn't work when it is in the url. 

I've made changes that work with my provider, and are hopefully backwards-compatible with anyone already using the plugin. I'm planning on submitting a pull request for your review.
